install/0000_90_cluster-autoscaler-operator_01_prometheus_rbac.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Why are this file and the ServiceMonitor file named differently from the others? There is an existing file with RBAC rules.